particle/particle/kinematics.py

```diff
@@ -0,0 +1,52 @@
+# Licensed under a 3-clause BSD style license, see LICENSE.
+"""
+Functions relevant to particle kinematics.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from hepunits.units import MeV, ns
+from hepunits.constants import hbar
+
+
+
+def width_to_lifetime(Gamma):
+    """
+    Convert from a particle decay width to a lifetime.
+
+    Parameters
+    ----------
+    Gamma : float > 0
+        Particle decay width, typically in MeV (any HEP energy unit is OK).
+
+    Returns
+    -------
+    Particle lifetime, in the HEP standard time unit ns.
+    """
+
+    if Gamma <= 0.:
+        raise ValueError( 'Input provided, %s <= 0!'.format(Gamma) )
+
+    # Just need to first make sure that the width is in the standard unit MeV
+    return hbar / float(Gamma / MeV)
```

Comment:
Like I said in the PR thread, the division by MeV is meant to be explicit, to help the reader. I also think the time spent on this is irrelevant for 99.999% of the cases. But if you think it's better to just make a comment and remove "/MeV", that can be done ...

Reply:
I think it's okay. But it would be good to either do it everywhere or not at all. A good test would be to change the base unit factors in one of the tests, then see if the results stay the same.

Reply:
Agreed. I will try and see where elsewhere one can use `hepunits` stuff ...

As for the test with a change of base unit factors, that's something that requires an upgrade in hepunits. Effectively you are asking for a scaling factor across all the units and subunits defined. That's something I believe `numericalunits` could be good for. As said on the org email exchanges, I will try and investigate this package ...

Reply:
I don't understand either comments, I'm afraid. I did get it that you mean a test. But aren't the examples I added exactly that test? Else I gather the following: in HEP units, MeV=1, GeV=1000, etc. Now, if I scale all these XeV by some factor and the units of time by 1/factor, all should be invariant. But there is more to do to keep the whole system of units in sync.

I agree that the first docstrings could be misleading. Now we say: you give as a time in ns and we return you the decay width in MeV, and vice-versa for the other function. That is clear.

I don't see what you suggest to do in sys.modules.

Reply:
I'm not sure it will work, I'll play with it in a few days.

Reply:
Cool. We are sure to be discussing these matter more … Thanks!